sys/newlib_syscalls_default/syscalls.c

```diff
@@ -149,7 +149,9 @@ static const struct heap heaps[NUM_HEAPS] = {
  */
 void _init(void)
 {
-    /* nothing to do here */
+    if (IS_USED(MODULE_FMT)) {
+        setvbuf(stdout, NULL, _IONBF, 0);
```

Comment:
How bad would it be if we drop this?

Reply:
We will get corrupted stdio in an unpredictable matter and tests will fail due to it :/